### DISCUSSION TITLE
Server is unable to remove some cached next/image files

### DISCUSSION THREAD

**Discussion by ericvrp** [vercel/next.js#20309] [Help]
# Bug report

## Describe the bug

next/image is unable to remove some cached files and doesn't serve the image to the client 

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

```
[Error: ENOENT: no such file or directory, unlink '/home/site/wwwroot/.next/cache/images/E+6+mv6oYa4ANIzZ0kiaerfeySRH03cAVDt2WaG11Ag=/1608313906485.VPLvDKLBAdkh+pqgGlEbDaFbnrh10o1grxHBUQLKd5o=.webp'] {
2020-12-18T17:56:57.645546847Z   errno: -2,
2020-12-18T17:56:57.645554347Z   code: 'ENOENT',
2020-12-18T17:56:57.645559847Z   syscall: 'unlink',
2020-12-18T17:56:57.645565347Z   path: '/home/site/wwwroot/.next/cache/images/E+6+mv6oYa4ANIzZ0kiaerfeySRH03cAVDt2WaG11Ag=/1608313906485.VPLvDKLBAdkh+pqgGlEbDaFbnrh10o1grxHBUQLKd5o=.webp'
2020-12-18T17:56:57.645571848Z }
```

## Screenshots

There is a background image visible in this screenshot but the one where the name Michelle.. is written does not show because of the error above.

![image](https://user-images.githubusercontent.com/4216292/102645703-2a2b9880-4163-11eb-9b9c-6fc1d2a871cb.png)

## System information

Deployed to Azure

**Comment by jacobhytter** [comment]
Ill get the same error with my site on Prod


[Error: EBUSY: resource busy or locked, unlink '/usr/src/app/.next/cache/images/lRPOoujTwYy82c49yjV79GL3qLqzbgKYSkD3uTnd+1w=/.nfs9d57298709acc64500000001'] {
errno: -16,
code: 'EBUSY',
syscall: 'unlink',
path: '/usr/src/app/.next/cache/images/lRPOoujTwYy82c49yjV79GL3qLqzbgKYSkD3uTnd+1w=/.nfs9d57298709acc64500000001'

**Comment by kobars** [comment]
I get the same error

[Error: ENOENT: no such file or directory, unlink '/app/.next/cache/images/20++lFeeEifDtTQLBqbj+5o1vXda9LpGMwL6RK2yDgo=/1625283188897.uvGo5iJqflMTHljefe8+1Lbl9SO7XpExXxyaLzKQERA=.webp'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'unlink',
  path: '/app/.next/cache/images/20++lFeeEifDtTQLBqbj+5o1vXda9LpGMwL6RK2yDgo=/1625283188897.uvGo5iJqflMTHljefe8+1Lbl9SO7XpExXxyaLzKQERA=.webp'
}

**Comment by Haris-Rahat-Dev** [comment]
I get the same issue , and it tells me that it can't load this image, but the image is in the cache folder inside .next.
